SConstruct

```diff
@@ -279,9 +280,11 @@ def scons():
     buildinfo = prereqs.get_build_info()
     buildinfo.gen_script('.build_vars.sh')
     buildinfo.save('.build_vars.json')
-    env.InstallAs("$PREFIX/TESTING/.build_vars.sh", ".build_vars.sh")
-    env.InstallAs("$PREFIX/TESTING/.build_vars.json", ".build_vars.json")
-    env.InstallAs("$PREFIX/lib/daos/VERSION", "VERSION")
+    daos_build.install(env, "lib/daos/TESTING/",
+                       ['.build_vars.sh', '.build_vars.json'])
+    # also install to $PREFIX/lib to work with existing avocado test code
+    daos_build.install(env, "lib/", ['.build_vars.sh', '.build_vars.json'])
```

Comment:
Why are these `.build_vars.*` files being installed in more than one location?

Reply:
Same reason: needed by cart.